torch/csrc/autograd/python_anomaly_mode.cpp

```diff
@@ -28,6 +84,45 @@ void PyAnomalyMetadata::store_stack() {
 }
 
 void PyAnomalyMetadata::print_stack(const std::string& current_node_name) {
+  // If we have a CapturedTraceback (mixed_stack mode), symbolize and print it
+  if (captured_traceback_) {
+    std::vector<torch::CapturedTraceback*> to_symbolize = {
+        captured_traceback_.get()};
+    torch::SymbolizedTracebacks symbolized = torch::symbolize(to_symbolize);
+
+    std::vector<std::string> filtered_frames;
+    for (uint64_t frame_idx : symbolized.tracebacks.at(0)) {
+      const auto& frame = symbolized.all_frames.at(frame_idx);
+      // Use the same filtering logic as MemoryViz.js frameFilter
+      if (shouldOmitFrame(frame.funcname, frame.filename)) {
+        continue;
+      }
+      std::ostringstream frame_oss;
+      frame_oss << "  File \"" << frame.filename << "\", line " << frame.lineno
+                << ", in " << frame.funcname;
+      filtered_frames.push_back(frame_oss.str());
+    }
+
+    // Skip the first 3 frames as they are internal to anomaly mode
+    // (store_stack, Node constructor, etc.)
+    size_t start_idx = std::min(static_cast<size_t>(3), filtered_frames.size());
+
+    std::ostringstream oss;
+    for (size_t i = start_idx; i < filtered_frames.size(); ++i) {
+      oss << filtered_frames[i] << "\n";
+    }
```

Comment:
So I've been endlessly annoyed that this particular string formatting function isn't easily accessible in C++, can you please put it in an easily accessible spot so I can use this in other places too.